.github/workflows/config.yml

```diff
@@ -1,6 +1,8 @@
-name: Node.js CI
+name: Run Harness Tests
```

Comment:
did you rebase on main? looks like this isn't supposed to be here

Reply:
maybe i did it before that got merged in

Reply:
fixed